specs/interop/predeploys.md

```diff
@@ -214,15 +232,15 @@ function relayMessage(uint256 _destination, uint256 _source, uint256 _nonce, add
       tstore(CROSS_DOMAIN_MESSAGE_SOURCE_SLOT, _source)
     }
 
+    sentMessages[messageHash] = true;
+
     bool success = SafeCall.call({
        _target: _target,
        _value: msg.value,
        _calldata: _message
     });
 
     require(success);
```

Comment:
should the call revert or split into an if clause based on the success of the call and emit two different events? 
something like this:
```
        bool success = _callWithAllGas(_target, _message);

        if (success) {
            successfulMessages[messageHash] = true;
            emit RelayedMessage(messageHash);
        } else {
            emit FailedRelayedMessage(messageHash);
        }
```
it's what we have in the code right now, so want to double check if we should stick to this or change it to match this part of the spec